src/components/Modals/SetAclPermissionModal.jsx

```diff
@@ -0,0 +1,200 @@
+// React Imports
+import React, { useContext, useState } from 'react';
+// Custom Hook Imports
+import { useSession, useStatusNotification } from '@hooks';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import SettingsIcon from '@mui/icons-material/Settings';
+import TextField from '@mui/material/TextField';
+import Typography from '@mui/material/Typography';
+// Utility Imports
+import { runNotification, setDocAclPermission } from '@utils';
+// Context Imports
+import { SignedInUserContext } from '@contexts';
+// Component Imports
+import DocumentSelection from '../Form/DocumentSelection';
```

Comment:
The imports in Line 22 and 23 could be simplified into `import { DocumentSelection, FormSection } from './Form'` given we're using the `index.js` file from `src/components/Form`.